lua/schemastore/catalog.lua

```diff
@@ -1623,7 +1623,7 @@ M.json = {
       description = "Bitbucket Pipelines CI/CD manifest",
       fileMatch = { "bitbucket-pipelines.yml" },
       name = "bitbucket-pipelines",
-      url = "https://bitbucket.org/atlassianlabs/intellij-bitbucket-references-plugin/raw/master/src/main/resources/schemas/bitbucket-pipelines.schema.json"
```

Comment:
This schema hosted in the repo of Bitbucket Linky plugin for IntelliJ is deprecated and will eventually be removed (I'm the maintainer of that plugin).